graphql/database/base/test-utils/ExampleRedisModel.js

```diff
@@ -0,0 +1,62 @@
+import types from '../../fieldTypes'
+import RedisModel from '../RedisModel'
+
+class ExampleRedisModel extends RedisModel {
+  static get name() {
+    return 'Example'
+  }
+
+  static get hashKey() {
+    return 'id'
```

Comment:
Related to other comments, will this model still work with something other than `id` here?

Reply:
After the most recent updates, this model should still work with something other than id.